### ISSUE TITLE
Title - Description after moving to YPTStorage

### ISSUE THREAD

**Issue by GreenDragonTavern** [WWBN/AVideo-Storage#30]
After moving video to YPTStorage, Title and Description change, why? Images included and only thing that changed was moving from local to remote storage.

![LocalStorage](https://user-images.githubusercontent.com/85363361/168609371-3565901e-3417-485f-bcc0-fc651cbabe4a.jpg)
![YPTStorage](https://user-images.githubusercontent.com/85363361/168609400-91b07a6e-c57f-44cd-8a0d-84d02b9f40b1.jpg)

**Comment by DanielnetoDotCom** [comment]
the storage move does not suppose to make it happen. maybe you are outdated, and this issue is already fixed

but I saw a similar thing happen with the clone plugin before.

and I build it to help fix it.

https://github.com/WWBN/AVideo/blob/master/install/fixSlashN.php

this is a command-line script that you can also modify to fix the multiple slashes

**Comment by DanielnetoDotCom** [comment]
I just sent a modification to the fixSlashN.php file, I hope it works for you

**Comment by GreenDragonTavern** [comment]
I am definitely current on all; AVideo, Encoder, and Storage. Original video stored locally was spot on, no issues, and all I did was move to YPTStorage. Is the multiple slash fix implemented on move to YPTstorage? Just did a test on a YouTube video that stored it directly on YPTStorage and the multiple slash issue is definitely there. Changed back to local storage and presto, no slash issue on same video.

**Comment by DanielnetoDotCom** [comment]
can you reproduce the slash problem on demo site?

if works there means you are not on the latest. or the error is something else

**Comment by GreenDragonTavern** [comment]
Just tested video on your site. It was stored local, and I moved it to storage 3. Yes, demo site is having same slash issue after moving to remote storage.

**Comment by GreenDragonTavern** [comment]
Both regarding YPTStorage. If I set YPTStorage as default and upload (meaning directly to YPTStorage) there's a slash issue. If I upload and store locally, there is no slash issue until after I move it to YPTStorage. That make sense?

**Comment by DanielnetoDotCom** [comment]
I did a test on the demo site and I could not reproduce it.

both (Moments w/ Michael Jaco~ Retired 35Year Navy Seal & CIA Op Plays Crystal Song Bowl on Casey Key, FL) were already with the "\n" before I transfer

and after the transfer, I did not see any difference

also, I transferred the “2000 Mules” Trailer and it seems to be fine

![chrome-capture-2022-4-16](https://user-images.githubusercontent.com/18009293/168657857-4feae6c2-9bb7-4756-b210-5ae3a4a81985.gif)

**Comment by DanielnetoDotCom** [comment]
where does this video come from?

is it a youtube video?

can you please share a link so I can make some tests?

**Comment by GreenDragonTavern** [comment]
It's a YouTube video, here's link: https://youtu.be/XgB64_vJn1w

**Comment by GreenDragonTavern** [comment]
Here's a really good example:  https://youtu.be/ay_Eohsjn2o

Upload it from YouTube, local only. Look at title, description, it's exactly as is as seen on YouTube. Then transfer to one of your YPTStorage sites and see all the //////////////////////////////////////////

**Comment by DanielnetoDotCom** [comment]
Hi, I guess the last AVideo update fix the issue

**Comment by GreenDragonTavern** [comment]
This issue is definitely not fixed using paid plugin YPTStorage.
![AfterLatestPull](https://user-images.githubusercontent.com/85363361/168686243-00e341c7-116a-444c-8dbc-2845fe9031a9.jpg)

**Comment by DanielnetoDotCom** [comment]
Did you test on demo?

Did you update your avideo with git pull?

**Comment by GreenDragonTavern** [comment]
Yes, and Yes. Your demo site does exactly the same as my site. Upload a YouTube video and storage set to local, works fine and Title, Description are perfect. Move that video to YPTStorage and //// shows up. Change default storage to YPTStorage and upload same video, //// shows up. It does not matter if you move to YPTStorage or natively store to YPTStorage it is the same result, you get ////.

I have tested twice on your demo site, the behavior was the same both times, and is exactly the same behavior happening on my site.

**Comment by DanielnetoDotCom** [comment]
I did a test and it was working 

I am away from pc today I will check it again tomorrow

**Comment by GreenDragonTavern** [comment]
Sounds like a plan, and thank you.

**Comment by DanielnetoDotCom** [comment]
I just make more tests on the demo site.

I moved the video between storages

local to site, site to site, and site to local

also I cleared the cache to make sure it was working.

everything works as expected .

Sorry I am not able to reproduce this issue.

**Comment by GreenDragonTavern** [comment]
Download video 2 times (very small clip): https://youtu.be/kYJ82izYLmc

Keep one on local storage, and move one to any YPTStorage device on your demo system. Edit the one on local storage, it is exactly as seen on YouTube. Edit the one you moved to YPTStorage, it is a mess...

**Comment by GreenDragonTavern** [comment]
https://demo.avideo.com/video/384/moments-w-michael-jaco-~-are-you-familiar-with-turtle-island

https://demo.avideo.com/video/383/moments-w-michael-jaco-~-are-you-familiar-with-turtle-island-1

**Comment by DanielnetoDotCom** [comment]
I see, the error happens when you edit a moved video?

**Comment by GreenDragonTavern** [comment]
No, nothing to do with editing Title, Description. Just replicated error on your demo site, it's there now...one local, one YPTStorage.

![YPTStorage_Demo2](https://user-images.githubusercontent.com/85363361/168874799-101376a6-790c-4767-8e77-1195114acd24.jpg)

**Comment by GreenDragonTavern** [comment]
And this is what displays for video on local storage:

![LocalStorage_Demo2](https://user-images.githubusercontent.com/85363361/168875539-f405c86a-580b-4e44-8d17-1de24efdf822.jpg)

**Comment by DanielnetoDotCom** [comment]
Do you mind recording a video reproducing the error?

I am still having a hard time to reproduce it

**Comment by GreenDragonTavern** [comment]
I am heading to our data center shortly...will make a recording of process later today. My time is UTC-8, and you?

**Comment by DanielnetoDotCom** [comment]
I am in GMT -3

I will probably be able to check it tomorrow only
